pyrightconfig.json

```diff
@@ -22,4 +22,7 @@
     "reportUndefinedVariable": "warning",
     "reportOptionalOperand": "warning",
     "reportMissingImports": "warning",
+    "reportPrivateImportUsage": "warning",
+    // Suppress because it flags our standard pattern @staticmethod __classcall__(cls, ...)
```

Comment:
comments in json files are sadly not possible. the common workaround I've seen is to add them as another field, i.e. `reportSelfClsParameterName.comment` (not sure if it works here or if pyright complains)

Reply:
It's a Microsoft extension ("JSONC"), see https://github.com/microsoft/node-jsonc-parser